app/models/user.rb

```diff
@@ -54,7 +54,7 @@ def create_personal_namespace!
     Namespace.find_or_create_by!(
       team:     team,
       name:     username,
-      registry: Registry.last # TODO: fix once we handle more registries
+      registry: Registry.get
```

Comment:
do you want to add the same comment to all places where you were using Registry.last/first?